### ISSUE TITLE
AAPB content on Open Vault

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/openvault3#418]
A total of 1,331 programs on AAPB are missing from Open Vault.

-- There are only 171 total records for WGBH Forum Network on Open Vault. On AAPB, we have 672 WGBH Forum Network programs available online. None of the records on Open Vault are in the "Available Online" facet.

-- There are only 7 records for The First Amendment on Open Vault. On AAPB we have 181 First Amendment programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There are only 17 records for Listen Here on Open Vault. On AAPB we have 49 Listen Here programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There are 358 WGBH Journal on Open Vault. On AAPB we have 147 WGBH Journal programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There are only 4 records for Long, Hot Summer '64 on Open Vault. On AAPB we have 5 Long Hot Summer, 64 programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There is only 1 record for United Nations Security Council on Open Vault. On AAPB we have 50 United Nations Security Council programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There are zero records for United Nations General Assembly on Open Vault. On AAPB we have 17 United Nations General Assembly programs available online. 

--There are only 4 records for Crock Snow Reports from Germany on Open Vault. On AAPB we have 29 Crock Snow Reports from Germany programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There is only 1 record for Vietnam War Report on Open Vault. On AAPB we have 31 Vietnam War Report programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There 1 record for Public Television Hearings on Open Vault. On AAPB we have 26 Public Television Hearings programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There is only 1 record for Boston College Citizenship Series on Open Vault. On AAPB we have 13 Boston College Citizenship Series programs available online. None of the records on Open Vault are in the "Available Online" facet.

--There are zero records for Framingham Community Forum on Open Vault. On AAPB we have 14 Framingham Community Forum programs available online. 

--There are 4 Sunday Forum on Open Vault. On AAPB we have 43 Sunday Forum programs available online. None of the records on Open Vault are in the "Available Online" facet. None of the records on Open Vault are in the "Available Online" facet.

--There are zero records for United States Senate Foreign Relations Committee on Open Vault. On AAPB we have 26 United States Senate Foreign Relations Committee programs available online. 

--There is one record for Byline on Open Vault. On AAPB we have 13 Byline programs available online. None of the records on Open Vault are in the "Available Online" facet. None of the records on Open Vault are in the "Available Online" facet.

--There are two WGBH Roundtable programs on Open Vault. On AAPB we have 15 WGBH Roundtable programs available online. None of the records on Open Vault are in the "Available Online" facet. None of the records on Open Vault are in the "Available Online" facet.

**Comment by caseyedavis12** [comment]
This is not a blocker for launch, but I will be working on harmonization after launch.

**Comment by mccalluc** [comment]
@Muraszko : This is something other than records not being ingested because of validation problems?

**Comment by Muraszko** [comment]
@mccalluc 
Yes, this is a metadata thing.  There is opportunity to list additional AAPB outside URLs in OV records.
